jenkins-build-pack-api/Jenkinsfile

```diff
@@ -103,12 +74,21 @@ node() {
 		}
 
 		if(configObj.service_id){
-			config = serviceConfigdata.loadServiceMetadata(configObj.service_id,configLoader)
+			config = serviceMetadataLoader.loadServiceMetadata(configObj.service_id)
 		}else{
 			error "Service Id is not available."
 		}
 	}
-
+	
+    if(!events) { error "Can't load events module"	} //Fail here
+    events.setUrl(g_base_url + "jazz/events")
```

Comment:
Can you please create one initialize method and pass all these params as part of that?